BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.kt

```diff
@@ -396,7 +398,11 @@ open class BrickDataManager : Serializable {
      * Method called to release any related resources.
      */
     open fun onDestroyView() {
-        recyclerView?.adapter = null
+        try {
+            recyclerView?.adapter = null
```

Comment:
this line doesn't look necessary!

recyclerView?.adapter contains a reference to brickRecyclerAdapter, and we set it to null on line 406.

Reply:
The ticket has this fix listed as the solution. The crash is intermittent and not worth digging deeper. We should just catch the exception here when it does get thrown. See this discussion here: https://github.com/wayfair/brickkit-android/issues/259

Reply:
❤️ for the review though 😃